android/src/main/java/com/swmansion/reanimated/layoutReanimation/SharedTransitionManager.java

```diff
@@ -155,11 +189,43 @@ private boolean tryStartSharedTransitionForViews(
     setupTransitionContainer();
     reparentSharedViewsForCurrentTransition(sharedElements);
     orderByAnimationTypes(sharedElements);
+    return true;
+  }
+
+  protected void onScreenWillDisappear() {
+    if (!mIsTransitionPrepared) {
+      return;
+    }
+    mIsTransitionPrepared = false;
+    for (var sharedElement : mSharedElementsWithAnimation) {
```

Comment:
In our codebase we mostly use concrete types instead of `var`, is this a deliberate decision to use `var` here?